ui/src/containers/AppDetailsContainer.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
```

Comment:
Is there much difference in the two? would it be possible to resuse the other container as well in some way?

Reply:
Good point, and my thoughts too. At the moment, the state data is different and also the api call

Reply:
Could we pass those as props somehow?

Reply:
That was my initial thinking, and exploring whether this may be a reasonable possibility

Reply:
@laurafitzgerald went with a separate container for the AppDetailsView table. Possible to re-use existing container by editing it to accept props and render a table of all apps, or a table of app details. 

Optimising now though at projects infancy where the UI may evolve a lot yet may make adding further similar views with tables more complex, thus why I went with an additional container here rather than abstract prematurely yet. 

If you disagree let me know. I think we can always abstract later if the UI evolves and the patterns to UI become more clear.